bioblend/galaxy/jobs/__init__.py

```diff
@@ -20,8 +20,9 @@ def __init__(self, galaxy_instance):
         self.module = 'jobs'
         super().__init__(galaxy_instance)
 
-    def get_jobs(self, state=None, tool_id=None, user_details=False,
-                 date_range_min=None, date_range_max=None, history_id=None):
+    def get_jobs(self, state=None, tool_id=None, user_details=False, user_id=None,
+                 limit=500, offset=0, date_range_min=None, date_range_max=None,
+                 history_id=None, workflow_id=None, invocation_id=None):
```

Comment:
Given all these parameters are still not in any BioBlend release, should we rearrange them in some logical order? E.g. state, history_id, invocation_id, tool_id, workflow_id, user_id, date_range_min, date_range_max, limit, offset, user_details